spec/csharp/kaitai_struct_csharp_tests/CommonSpec.cs

```diff
@@ -1,9 +1,14 @@
 ﻿using System;
+using System.IO;
 
 namespace Kaitai
 {
     public class CommonSpec
     {
-        public String SRC_DIR = "src/";
+        public String SourceFile(string filename)
+        {
+            var currentDir = Path.GetDirectoryName(typeof(CommonSpec).Assembly.Location);
+            return Path.Combine(currentDir, "src", filename);
+        }
```

Comment:
I don't quite get what it does, but if it tries to derive location of binary input files (i.e. `src`) from the location of assembly somehow, probably it's a bad idea. That `src` is usually located in `tests/src`, while assembly ends up in `tests/spec/csharp/kaitai_struct_csharp_tests/bin/Debug/kaitai_struct_csharp_tests.dll`.

In ideal world, we probably shall supply the location of test input files using some external mechanism, like environment variable, interpreter property, command line arguments or something like that. Test running automation scripts (like `tests/run-csharp`) actually have the config (`tests/config`) which should be the only place which actually stores path to `src`, and they should supply this path to the tests as a piece of external config.

`src` was a lousy naming choice, by the way. Naming test inputs as "sources" in a system that deals with generation of source files from other source files, was, well, a bad idea of mine :) Hope to fix it someday.